packages/web-components/src/utils/dom.spec.ts

```diff
@@ -0,0 +1,47 @@
+import { quicksort } from './dom';
+
+describe('quicksort', () => {
```

Comment:
I'd like to see some more tests here. What happens when the text inside the cells is a number? What if you have a mix of cells with letters and numbers? What happens if the text includes a symbol or starts with a symbol?

Reply:
These are great, thanks for adding all the tests!